TimberAPI/UIBuilderSystem/PresetSystem/ButtonPresetFactory.cs

```diff
@@ -0,0 +1,637 @@
+                .AddClass(TimberApiStyle.Buttons.Normal.SpeedButton0)
+                .AddClass(TimberApiStyle.Buttons.Hover.SpeedButton0Hover)
+                .AddClass(TimberApiStyle.Buttons.Active.SpeedButton0Active)
+                .AddClass(TimberApiStyle.Sounds.Click)
+                .SetName(name)
+                .SetHeight(size == default ? new Length(24, Pixel) : size)
+                .SetWidth(size == default ? new Length(24, Pixel) : size);
+            builder?.Invoke(button);
+            return button.Build();
+        }
+        
+        public LocalizableButton SpeedButton1(string name = null, Length size = default, Action<ButtonBuilder> builder = default)
+        {
+            ButtonBuilder button = _componentBuilder.CreateButton()
+                .AddClass(TimberApiStyle.Buttons.Normal.SpeedButton1)
+                .AddClass(TimberApiStyle.Buttons.Hover.SpeedButton1Hover)
+                .AddClass(TimberApiStyle.Buttons.Active.SpeedButton1Active)
+                .AddClass(TimberApiStyle.Sounds.Click)
+                .SetName(name)
+                .SetHeight(size == default ? new Length(24, Pixel) : size)
+                .SetWidth(size == default ? new Length(24, Pixel) : size);
+            builder?.Invoke(button);
+            return button.Build();
+        }
+        
+        public LocalizableButton SpeedButton2(string name = null, Length size = default, Action<ButtonBuilder> builder = default)
+        {
+            ButtonBuilder button = _componentBuilder.CreateButton()
+                .AddClass(TimberApiStyle.Buttons.Normal.SpeedButton2)
+                .AddClass(TimberApiStyle.Buttons.Hover.SpeedButton2Hover)
+                .AddClass(TimberApiStyle.Buttons.Active.SpeedButton2Active)
+                .AddClass(TimberApiStyle.Sounds.Click)
+                .SetName(name)
+                .SetHeight(size == default ? new Length(24, Pixel) : size)
+                .SetWidth(size == default ? new Length(24, Pixel) : size);
+            builder?.Invoke(button);
+            return button.Build();
+        }
+        
+        public LocalizableButton SpeedButton3(string name = null, Length size = default, Action<ButtonBuilder> builder = default)
+        {
+            ButtonBuilder button = _componentBuilder.CreateButton()
+                .AddClass(TimberApiStyle.Buttons.Normal.SpeedButton3)
+                .AddClass(TimberApiStyle.Buttons.Hover.SpeedButton3Hover)
+                .AddClass(TimberApiStyle.Buttons.Active.SpeedButton3Active)
+                .AddClass(TimberApiStyle.Sounds.Click)
+                .SetName(name)
+                .SetHeight(size == default ? new Length(24, Pixel) : size)
+                .SetWidth(size == default ? new Length(24, Pixel) : size);
+            builder?.Invoke(button);
+            return button.Build();
+        }
+        
+        public LocalizableButton BugTracker(string name = null, Length size = default, Action<ButtonBuilder> builder = default)
```

Comment:
what is this

Reply:
Its a button, with a bug. the one for reporting bugs.